templates/docs/examples/patterns/buttons/alignment.html

```diff
@@ -0,0 +1,22 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Buttons / Alignment{% endblock %}
+
+{% block standalone_css %}patterns_buttons{% endblock %}
+
+{% block content %}
+<button>Classless button</button>
+<button>Classless button</button>
+
+<button class="p-button--neutral">Neutral button</button>
```

Comment:
Could you group these into separate paragraphs?

Reply:
Cool - I've updated